helm/csi-powerstore/values.yaml

```diff
@@ -34,6 +34,20 @@ externalAccess:
 # Default value: None
 imagePullPolicy: IfNotPresent
 
+# nfsAcls: enables setting permissions on NFS mount directory
+# This value acts as default value for NFS ACL (nfsAcls), if not specified for an array config in secret
+# Permissions can be specified in two formats:
+#   1) Unix mode (NFSv3)
+#   2) NFSv4 acl (NFSv4)
+# Allowed values:
+#   1) Unix mode: valid octal mode number
+#      Examples: "0777", "777", "0755"
+#   2) NFSv4 acls: valid NFSv4 acls, seperated by comma
```

Comment:
Please mention that only NFSv4 ACLs are supported

Reply:
done